code/__HELPERS/constants.dm

```diff
@@ -1,2 +1,2 @@
-var/global/const/TICKS_IN_DAY = 864000
-var/global/const/TICKS_IN_SECOND = 10
+var/const/TICKS_IN_DAY = 864000
```

Comment:
But it's exactly how it works. If the var wasn't global, a second world object would have a second instance of this var.